ecommerce/models.py

```diff
@@ -78,6 +79,10 @@ class ProductVersion(TimestampedModel):
     class Meta:
         indexes = [models.Index(fields=["created_on"])]
 
+    def save(self, *args, **kwargs):  # pylint:disable=arguments-differ
+        super().save(*args, **kwargs)
+        task_helpers.sync_hubspot_product(self)
```

Comment:
Could we stay away from overriding `save()` and use signals instead? This burnt us several times in discussions and I'd hate to head down that path again.

Reply:
👍